src/graphics/program-lib/chunks/combineDiffuseSpecularOld.frag

```diff
@@ -1,4 +1,3 @@
 vec3 combineColor() {
     return mix(dAlbedo * dDiffuseLight + dSpecularLight * dSpecularity, dReflection.rgb, dReflection.a);
 }
```

Comment:
Just revert this file. You actually want the newline at the end because it formats constructed shaders better anyway.

Reply:
reverted